adapters/handlers/grpc/v1/parse_aggregate_request.go

```diff
@@ -82,6 +86,233 @@ func (p *AggregateParser) Aggregate(req *pb.AggregateRequest) (*aggregation.Para
 		params.Filters = filter
 	}
 
+	// targetCombination is not supported with Aggregate queries
```

Comment:
Do you think `targetCombination` will ever be supported for Aggregate or does the concept not apply?

Reply:
IMO there's nothing stopping us from supporting this in gRPC API